src/pages/PagesRouter.tsx

```diff
@@ -15,6 +15,7 @@ import { status } from '@selectors/solanaWallet'
 import { SinglePositionPage } from './SinglePositionPage/SinglePositionPage'
 import { pools } from '@selectors/pools'
 import Footer from '@components/Footer/Footer'
+import IDO from './IDOPage/IDO'
```

Comment:
task was only for ui, revert changes here

Reply:
done